src/integrations/stripe.js

```diff
@@ -0,0 +1,101 @@
+import { loadStripe } from '@stripe/stripe-js';
```

Comment:
the other integrations are all loaded async rather than being installed in package.json. can we not do that here?

Reply:
Stripe-js is a very light weight wrapper lib that does really one thing and that is load stripe.js from a secure, PSI-compliant location.  Yes, we could duplicate that and I looked into doing that, but it is not trivial and why re-invent the wheel here so this is the way I went.  The uncompressed size of stripe-js is 3.4k.  Quite small.

Reply:
BTW, the actual loading of stripe.js (the larger and secure one) is done async and only if the integration is active.

Reply:
ok cool, I saw that it was importing loadStripe but wasn't 100% sure if that was happening. I think I'm ok with this if @bo-dun-1 is